ui/index.html

```diff
@@ -29,10 +29,14 @@
       }
 
       .buffer-container {
-        height: calc(100% - 40px);
+        height: calc(100% - 40px - 24px);
```

Comment:
Don't you want text-height-ish measurements here? Unsure.

```suggestion
        height: calc(100% - 40px - 1em);
```

Reply:
do you mean just to make it accurate to the text? otherwise I'd say

```suggestion
        height: calc(100% - 16px);
```

either way webterm seems to use px values everywhere

Reply:
Played with both of these, @arthyn I think youre dloing PEMDAS wrong, it should be 54px, I used the 1em version